installers/python-ci/radiasoft-download.sh

```diff
@@ -43,10 +44,11 @@ python_ci_main() {
             set -x
```

Comment:
This is on line 42 (`su - vagrant`). The EOF2 should be in single quotes because `\$x` will get interpolated twice, and we only want it to be once.

Reply:
085af5b7bc7474e434c3f7ad7cc88ba32ed8cbb8